common/src/View/CreateSimpleBrushTool.cpp

```diff
@@ -37,8 +38,17 @@ namespace TrenchBroom {
             auto document = kdl::mem_lock(m_document);
             const auto game = document->game();
             const auto builder = Model::BrushBuilder(document->world(), document->worldBounds(), game->defaultFaceAttribs());
-            auto* brush = document->world()->createBrush(builder.createCuboid(bounds, document->currentTextureName()));
```

Comment:
I think you should intersect the given bounds with worldbounds instead. The exceptions will go a way with my next PR anyway.

Reply:
We don't support brushes touching the world bounds (there was an issue about this recently).
Also there are other reasons it can throw, so what I have is the correct approach imo.

Reply:
Ok, I'll refactor it to use the new error handling in my PR then.